app/src/renderer/vuex/modules/node.js

```diff
@@ -44,7 +44,11 @@ export default function({ node }) {
         rootState.wallet.zoneIds.unshift(header.chain_id)
       }
 
-      await dispatch("maybeUpdateValidators", header)
+      // updating the header is done even while the user is not logged in
```

Comment:
why do we do this?

Reply:
yes, we bind to the "new header" events before signing in. the event would then trigger an update of the validatorset which is skipped here. This will also change when we react to the validatorset updated event.